### ISSUE TITLE
Incorrect numbers on positional parameters in queries-generating-most-temporary-files

### ISSUE THREAD

**Issue by oskarwiksten** [dalibo/pgbadger#264]
![screenshot - 01292016 - 02 15 49 pm](https://cloud.githubusercontent.com/assets/1265647/12676552/8d1fd5fa-c693-11e5-84b9-925eeba42fd5.png)
(problem in above screenshot is that many parameters are named `$10` when they should be named `$11`, `$12` etc)

To reproduce:
* Tested with Pgbadger 7.3
* Have a query that has positional parameters (named `$1`, `$2`, .. etc)
* Have more than 10 of these positional parameters
* Display the query in "Queries generating the most temporary files (N)"
* Notice how the parameters that were originally named, for example, `$11` and `$12` are now displayed as `$10`.

Example data to reproduce:
```bash
$ cat input
2016-01-25 23:00:36 UTC [19847]: [3-1] user=a,db=b LOG:  duration: 438.900 ms  execute <unnamed>: SELECT id, created FROM account WHERE (id IN ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20))
2016-01-25 23:00:36 UTC [19847]: [4-1] user=a,db=b DETAIL:  parameters: $1 = '1', $2 = '2', $3 = '3', $4 = '4', $5 = '5', $6 = '6', $7 = '7', $8 = '8', $9 = '9', $10 = '10', $11 = '11', $12 = '12', $13 = '13', $14 = '14', $15 = '15', $16 = '16', $17 = '17', $18 = '18', $19 = '19', $20 = '20'
2016-01-25 23:00:36 UTC [19847]: [5-1] user=a,db=b LOG:  temporary file: path "base/pgsql_tmp/pgsql_tmp19847.252", size 11352992

$ ./pgbadger -f stderr --prefix '%t [%p]: [%l-1] user=%u,db=%d ' -o output.html input

$ fgrep nu0 output.html
```

Output looks like:
```
$<span class="nu0">7</span><span class="sy0">,</span>
$<span class="nu0">8</span><span class="sy0">,</span>
$<span class="nu0">9</span><span class="sy0">,</span>
$<span class="nu0">10</span><span class="sy0">,</span>
$<span class="nu0">10</span><span class="sy0">,</span>
$<span class="nu0">10</span><span class="sy0">,</span>
$<span class="nu0">10</span><span class="sy0">,</span>
$<span class="nu0">10</span><span class="sy0">,</span>
$<span class="nu0">10</span><span class="sy0">,</span>
```

Expected output should be:
```
$<span class="nu0">7</span><span class="sy0">,</span>
$<span class="nu0">8</span><span class="sy0">,</span>
$<span class="nu0">9</span><span class="sy0">,</span>
$<span class="nu0">10</span><span class="sy0">,</span>
$<span class="nu0">11</span><span class="sy0">,</span>
$<span class="nu0">12</span><span class="sy0">,</span>
$<span class="nu0">13</span><span class="sy0">,</span>
$<span class="nu0">14</span><span class="sy0">,</span>
$<span class="nu0">15</span><span class="sy0">,</span>
```

Further troubleshooting: The mangling of these numbers happens in function `normalize_query`. The following code also reproduces the issue, in a shorter way:
```perl
print STDERR normalize_query('SELECT id, created FROM account WHERE (id IN ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20))');
```

**Comment by darold** [comment]
Thanks for the report and effort to find the source of the issue. It might be fixed now with last commit fe709c5.

Best regards,

**Comment by oskarwiksten** [comment]
Great, fe709c5 fixes the issue. Thanks @darold !
